src/auth/computeclient.ts

```diff
@@ -66,21 +66,20 @@ export class Compute extends OAuth2Client {
   protected async refreshTokenNoCache(refreshToken?: string|
                                       null): Promise<GetTokenResponse> {
     const tokenPath = `service-accounts/${this.serviceAccountEmail}/token`;
-    let res: AxiosResponse<CredentialRequest>;
+    let data: CredentialRequest;
     try {
-      res = await gcpMetadata.instance(tokenPath);
+      data = await gcpMetadata.instance(tokenPath);
     } catch (e) {
       e.message = 'Could not refresh access token.';
       throw e;
     }
-    const tokens = res.data as Credentials;
-    if (res.data && res.data.expires_in) {
-      tokens.expiry_date =
-          ((new Date()).getTime() + (res.data.expires_in * 1000));
+    const tokens = data as Credentials;
+    if (data && data.expires_in) {
+      tokens.expiry_date = ((new Date()).getTime() + (data.expires_in * 1000));
       delete (tokens as CredentialRequest).expires_in;
     }
     this.emit('tokens', tokens);
-    return {tokens, res};
+    return {tokens, res: null};
```

Comment:
This seems like a behavior change?

Reply:
Ah, good catch :) I actually hate the way we return the responses all over the place with this API.  There was a breaking change to `gcp-metadata` where we stripped that stuff out, so it makes it hard/impossible to show here.

Do you think this is enough of a change to merit a semver major release?

Reply:
Semver major is about communicating changes, and IMO the size of the change shouldn't have an impact on the semver-ness. If this is publicly exposed API, it would make sense to make it semver major.

Reply:
pingpingping @JustinBeckwith